.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 echo "Enable e2e-tests on release-1.3"
```

Comment:
does this pipeline ONLY run on one branch at a time? can we not us this for main AND release-1.3 in parallel?

Reply:
Yes, this pipeline runs only on one branch at a time thus, it doesn't run on main and release-1.3 in parallel.